src/include/OpenImageIO/dassert.h

```diff
@@ -68,7 +70,7 @@
 
 #ifndef ASSERT
 # define ASSERT(x)                                              \
-    ((x) ? ((void)0)                                            \
+    (OIIO_UNLIKELY(x) ? ((void)0)                               \
```

Comment:
Oh yeah, I got it backwards. 



On December 4, 2015 12:41:01 PM PST, Christopher Kulla <notifications@github.com> wrote:
>> @@ -68,7 +70,7 @@
>>  
>>  #ifndef ASSERT
>>  # define ASSERT(x)                                              \
>> -    ((x) ? ((void)0)                                            \
>> +    (OIIO_UNLIKELY(x) ? ((void)0)                               \
>
>Shouldn't asserts be "likely"?
>We expect them to always be true?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/OpenImageIO/oiio/pull/1279/files#r46729716

-- 
Larry Gritz
lg@larrygritz.com